Assignment_2/Index.html

```diff
@@ -0,0 +1,145 @@
+        "border-color": "green",
+        "border-style": "solid"
+      });
+      //Selects elements that have the specified attribute with a value either equal to a given string or starting with that string followed by a hyphen (-).
+      $("p[name|='marlabs']").css({
+        "border-width": "1px",
+        "border-color": "green",
+        "border-style": "solid"
+      });
+      //Selects elements that have the specified attribute with a value ending exactly with a given string. The comparison is case sensitive.
+      $("p[name$='marlabs']").css({
+        "border-width": "1px",
+        "border-color": "green",
+        "border-style": "solid"
+      });
+      //Selects elements that have the specified attribute with a value beginning exactly with a given string.
+      $("p[name^='marlabs']").css({
+        "border-width": "1px",
+        "border-color": "green",
+        "border-style": "solid"
+      });
+      //Select elements that either don’t have the specified attribute, or do have the specified attribute but not with a certain value
+      $("p[name!='marlabs']").css({
+        "border-width": "2px",
+        "border-color": "blue",
+        "border-style": "dotted"
+      });
+
+      $("button[name='on']").on("click", function() {
+        console.log("I am created by on function button");
+      });
+
+      $("button[name='one']").one("click", function() {
+        console.log("I am created by one function button");
+      });
+      $("button[name='trigger']").trigger("click");
+      $("button[name='trigger']").on("click", function() {
+        console.log("I am the child of Trigger function");
+      });
+      $("button[name='dblclick']").dblclick("click", function() {
+        console.log("I am created by double click function button");
+      });
+
+    $("input").hover(function(){
+    $("input").css("background-color", "orange");
+    console.log("hover is called");
+    });
+
+      $("input").keydown(function() {
+        $("input").css("background-color", "yellow");
+        console.log("keydown is called");
+      });
+      $("input").keyup(function() {
+        $("input").css("background-color", "green");
+        console.log("keyup is called");
+      });
+      $("input").keypress(function() {
+        $("input").css("background-color", "blue");
+        console.log("keypress is called");
+      });
+     
+     
+
+      $(window).resize(function() {
+        $("#log").append("<div>Handler for .resize() called.</div>");
+      });
+
+      $(window).resize(function() {
+        $("body").prepend("<div>" + $(window).width() + "</div>");
+      });
+
+      $("button")
+        .last()
+        .css("color", "red");
+
+        
+        $.ajax({
+        url: "https://pokeapi.co/api/v2/pokemon/ditto/",
+        type: 'GET',
+        dataType: 'json', // added data type
+        success: function(res) {
+            console.log(res);
+           var result=[];
+            for (x of res.game_indices)
```

Comment:
Can you try `.map()` approach?

Reply:
Yes Sure

Reply:
Finished

Please let me know what do you feel?